internal/dns/dns.go

```diff
@@ -1,143 +0,0 @@
-/*
```

Comment:
Should we remove the dns directory as well?

Reply:
it is removed